docs/elements/hx-file-tile/index.html

```diff
@@ -0,0 +1,90 @@
+---
+title: <hx-file-tile>
+also:
+  components/file-tiles: File Tiles
+  components/icons: Icons
+  elements/hx-file-icon: <hx-file-icon>
+  elements/hx-icon: <hx-icon>
+---
+{% extends 'element.njk' %}
+{% block content %}
+
+  <section>
+    <p>
+      The custom <code>{{page.title}}</code> element is compromised of an file icon,
+      file name, file size, and a delete button.
```

Comment:
Update description to document its use, rather than how it's composed.

Maybe...
```
The custom ... element describes the states of a downloadable asset.
```